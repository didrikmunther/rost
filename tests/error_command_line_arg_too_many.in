fn main(argc: int, argv: &&char, error_here: int) {

}