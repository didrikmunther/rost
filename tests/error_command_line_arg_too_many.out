  --> ./tests/error_command_line_arg_too_many.in:1:34 => CompilerError
1 | fn main(argc: int, argv: &&char, error_here: int) {
  |                                  ^^^^^^^^^^
  |                                  └─ Too many parameters for main function, expected maximum of 2
2 | 